addons/medical/stringtable.xml

```diff
@@ -4053,20 +4074,17 @@
             <English>Medical training</English>
             <Polish>Wyszkolenie medyczne</Polish>
             <Italian>Addestramento Medico</Italian>
+            <German>Sanitätsausbildung</German>
         </Key>
         <Key ID="STR_ACE_Medical_Attributes_isMedicalVehicle_Description">
             <English>Whether or not the object will be a medical vehicle.</English>
             <Polish>Czy pojazdy ma być pojazdem medycznym?</Polish>
             <Italian>L'oggetto in questione sarà un veicolo medico o meno.</Italian>
+            <German>Ist das Objekt ein Sanitätsfahrzeug?</German>
```

Comment:
Ich würde mich vor 3.5.1 daran machen, die "Fragezeichenversion" in allen Dateien zu entfernen.